src/line_plot.py

```diff
@@ -18,14 +18,17 @@ def generate_line_plot(data: pd.DataFrame, ycol: str):
     """
```

Comment:
You should add the 'ycol' to your parameter list in the docstring.

Reply:
Fixed!